examples/datasets/plot_iris_dataset.py

```diff
@@ -35,6 +38,10 @@
 x_min, x_max = X[:, 0].min() - 0.5, X[:, 0].max() + 0.5
```

Comment:
I would not select the two features here. I find it weird when it comes to the last section. We can wait the next seciont and select the first and second columns